plugin-taskbar/lxqttaskbutton.cpp

```diff
@@ -421,6 +421,7 @@ void LXQtTaskButton::activateWithDraggable()
     // in progress to allow drop it into an app
     raiseApplication();
     KWindowSystem::forceActiveWindow(mWindow);
+    xcb_flush(QX11Info::connection());
```

Comment:
We're trying to avoid X.org specific code. So I'm not a fan of this. Can't this be achieved in display server agnostic way?

Reply:
I don't imagine it can be, since it is probably an X.org specific issue.
You can take the fix or leave it; I don't care since I use qmpanel now anyway. This was one of the smallest and easiest bug fixes to backport, so I won't even bother trying to send all the others your way.